File: iraf/unix/boot/spp/rpp/rpprat/alldig.r
#-h-  alldig			  306  local   12/01/80  15:53:45
# alldig - return YES if str is all digits
   include  defs

   integer function alldig (str)
   character str (ARB)
   integer i

   alldig = NO
   if (str (1) == EOS)
      return
   for (i = 1; str (i) != EOS; i = i + 1)
      if (!IS_DIGIT(str (i)))
	 return
   alldig = YES
   return
   end

File: unix/boot/spp/rpp/rppfor/alldig.f
      integer function alldig (str)
      integer str (100)
      integer i
      alldig = 0
      if (.not.(str (1) .eq. -2))goto 23000
      return
23000 continue
      i = 1
23002 if (.not.(str (i) .ne. -2))goto 23004
      if (.not.(.not.(48.le.str (i).and.str (i).le.57)))goto 23005
      return
23005 continue
23003 i = i + 1
      goto 23002
23004 continue
      alldig = 1
      return
      end
